docparse/output_walkthrough.mdx

```diff
@@ -0,0 +1,174 @@
+```
+The second element we see is a caption. We get back a bounding box and the text content of the caption.
+
+### Extracting Text from the Document
+
+```json text.json
+{'type': 'Text',
+ 'bbox': [0.08857761158662684,
+  0.5770943936434659,
+  0.5373964197495404,
+  0.5928012917258523],
+ 'properties': {'score': 0.589328944683075,
+  '_element_index': 5,
+  'page_number': 2},
+ 'text_representation': 'The wreckage was retained for further examination. \n'}
+```
+The third element we see is a text element. We get back a bounding box and the text content of the element.
+
+### Extracting a Table from the Document
+
+```json table.json
+
+{ 
+ 'type': 'table',
+ 'bbox': [0.09319061279296875,
+  0.6873569003018466,
+  0.9122876694623162,
+  0.8174545010653409],
+ 'properties': {'score': 0.8106631636619568,
+  'title': None,
+  'columns': None,
+  'rows': None,
+  '_element_index': 2,
+  'page_number': 2},
+ 'text_representation': None,
+ 'table': {'cells': [{'content': 'Aircraft Make:',
+    'rows': [0],
+    'cols': [0],
+    'is_header': True,
+    'bbox': {'x1': 0.09705882369281045,
+     'y1': 0.6968232236868688,
+     'x2': 0.27310294133986934,
+     'y2': 0.7097020113257576},
+    'properties': {}},
+   {'content': 'MARC JONES',
+    'rows': [0],
+    'cols': [1],
+    'is_header': True,
+    'bbox': {'x1': 0.31151960800653594,
+     'y1': 0.6968232236868688,
+     'x2': 0.43984150343137257,
+     'y2': 0.7097020113257576},
+    'properties': {}},...
+}
+```
+The fourth  element we see is an table. We get back a bounding box and the table structure, which includes the cells of the table and their properties.
+
+#### Displaying the Table
+
+Here we display the table in clean markdown format below. We clean the column headers to make them a separate row in the table.
+```python display_table.py
+import pandas as pd
+pandas = tables_to_pandas(partitioned_file)
+# Let's display the pandas dataframe
+table = pandas[4][1]
+table = pd.concat([pd.DataFrame([table.columns], columns=table.columns), table], ignore_index=True)
+
+table.columns = ['' for col in table.columns] # Optionally reset the column headers
+
+table
+```
+The output is given below:
+
+|    |                           |                  |                           |          |
+|---:|:--------------------------|:-----------------|:--------------------------|:---------|
+|  0 | Aircraft Make:            | MARC JONES       | Registration:             | N512P    |
+|  1 | Model/Series:             | PITTS MODEL 12   | Aircraft Category:        | Airplane |
+|  2 | Amateur Built:            |                  |                           |          |
+|  3 | Operator:                 | M12 AVIATION LLC | Operating Certificate(s)  | None     |
+|    |                           |                  | Held:                     |          |
+|  4 | Operator Designator Code: |                  |                           |          |
+
```

Comment:
Include this in the same section as output, or put as a tutorial and link from the Output section.

Reply:
Done.